tests/manual/property/TestProperty.cpp

```diff
@@ -34,23 +34,101 @@
 #include "FaceliftProperty.h"
 #include "TestProperty.h"
 
-#define EXPECT_TRUE(cond) if (!(cond)) { qFatal("Expectation wrong:" #cond "%s", qPrintable(__PRETTY_FUNCTION__)); };
+#define EXPECT_TRUE(cond) if (!(cond)) { qFatal("Expectation wrong at %s:%i / " #cond " / function: %s ", qPrintable(__FILE__), __LINE__, qPrintable(__PRETTY_FUNCTION__)); };
+
+class SignalSpy : public QObject {
```

Comment:
Use QSignalSpy instead

Reply:
QSignalSpy does not seem to support what I need here.